client/me/purchases/manage-purchase/change-payment-method/index.tsx

```diff
@@ -46,14 +45,13 @@ function ChangePaymentMethod( {
 	const hasLoadedUserPurchases = useSelector( hasLoadedUserPurchasesFromServer );
 	const purchase = useSelector( ( state ) => getByPurchaseId( state, purchaseId ) );
 	const payment = useSelector( ( state ) => getByPurchaseId( state, purchaseId )?.payment );
-	const selectedSite = useSelector( getSelectedSite );
 	const { isLoading: isLoadingStoredCards } = useStoredPaymentMethods( { type: 'card' } );
 
 	const { isStripeLoading } = useStripe();
 
 	const isDataLoading =
 		! hasLoadedSites || ! hasLoadedUserPurchases || isLoadingStoredCards || isStripeLoading;
-	const isDataValid = purchase && selectedSite;
+	const isDataValid = purchase;
```

Comment:
This variable seems kinda irrelevant now. You could just use `! purchase` below and it should do the same thing, and likewise anywhere else this variable is used 😄

Reply:
Good point - had to fix another bit of logic, so updated this as well.